packages/teleport/src/components/NodeList/NodeList.tsx

```diff
@@ -96,6 +105,14 @@ export const renderAddressCell = ({ addr, tunnel }: Node) => (
   <Cell>{tunnel ? renderTunnel() : addr}</Cell>
 );
 
+function tunnelMatcher(
+  targetValue: any,
+  searchValue: string,
+  propName: keyof Node & string
+) {
+  return propName === 'tunnel' && targetValue && propName.includes(searchValue);
```

Comment:
makes more sense to me even though they do same thing

```suggestion
  return propName === 'tunnel' && targetValue && searchValue.includes(propName);
```

Reply:
nvm, i am wrong